persistence/src/test/java/cz/muni/fi/pa165/BottleDAOTest.java

```diff
@@ -112,6 +113,81 @@ public void updateBottleTest() {
         assertEquals(updatedBottle.getStickerID(), "ID9");
         assertTrue(updatedBottle.isToxic());
     }
+
+    @Test
+    public void getAllBottlesFromManufacturerFromDateWithOne() {
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setName("name");
+        manufacturerDAO.save(manufacturer);
+
+        BottleType bt = new BottleType();
+        bt.setName("Vodka Amundsen");
+        bt.setSize(BigDecimal.valueOf(1000));
+        bt.setType(AlcoholType.VODKA);
+        bt.setVolume(BigDecimal.valueOf(37.5));
+        bt.setManufacturedBy(manufacturer);
+        bottleTypeDAO.save(bt);
+
+        Bottle b = new Bottle();
+        b.setToxic(false);
+        b.setStickerID("ID1123");
+        b.setProduced(new Date());
+        b.setBottleType(bt);
+        bottleDAO.save(b);
+
+        List<Bottle> bottles = bottleDAO.getAllBottlesFromManufacturerFromDate(manufacturer, new Date(1));
+
+        assertEquals(bottles.size(), 1);
+    }
+
+    @Test
+    public void getAllBottlesFromManufacturerWithMany() {
+        Manufacturer original = createManufacturer("nameadsdas");
+        Manufacturer different = createManufacturer("differentfdsfdsdwq");
+        Manufacturer anotherDifferent = createManufacturer("differentfdsfdsdwqdas");
```

Comment:
Taketo mena su dost skarede, skus to prepisat aspon na manufacturer1, manufacturer2 atd.